pkg/driver/constants.go

```diff
@@ -102,6 +102,9 @@ const (
 	// TagKeyPrefix contains the prefix of a volume parameter that designates it as
 	// a tag to be attached to the resource
 	TagKeyPrefix = "tagSpecification"
+
+	// OutpostArn represents key for outpost's arn
+	OutpostArn = "outpostarn"
```

Comment:
np: `OutpostArnKey` and `"outpostArn"`

Reply:
Will change in next rev makes it much more clear

Reply:
Upon reviewing the code again outpostArn will not work as we are checking for all lower case values but I will change the key in next rev.